package.json

```diff
@@ -76,8 +88,20 @@
     "lint:fix": "npm run lint -- --fix",
     "build": "cross-env BABEL_ENV=commonjs babel src --out-dir lib --source-maps",
     "flow": "flow",
+    "prettier": "prettier --write ./**/*.{js,css}",
```

Comment:
Woah, does prettier format css too?

Reply:
yeah, I guess it's not really applicable here. enabled it for markdown